src/debugging.bib

```diff
@@ -0,0 +1,203 @@
+@article{lawrance2013foraging,
+ author = {Lawrance, Joseph and Bogart, Christopher and Burnett, Margaret and Bellamy, Rachel and Rector, Kyle and Fleming, Scott D.},
+ title = {How Programmers Debug, Revisited: An Information Foraging Theory Perspective},
+ journal = {IEEE Trans. Softw. Eng.},
+ issue_date = {February 2013},
```

Comment:
remove the extra information that shows up in the bibliography entries -- authors, title, venue, year, and for journal articles the vol/issue/page numbers are all that is necessary